main/guides/orchestration/txvsportfolio.md

```diff
@@ -0,0 +1,168 @@
+of the transactional flow:
+
+```js
+/**
+ * Example flow for a transactional contract:
+ *
+ * @param {ZCFSeat} seat - The user's seat, including the tokens to send.
+ * @param {string} chainName - Name of the destination chain.
+ * @param {string} destAddr - Destination address on the chain.
+ */
+export const sendIt = async (
+  orch,
+  { sharedLocalAccountP, log, zoeTools },
+  seat,
+  { chainName, destAddr }
+) => {
+  // 1. Extract the single token amount from the proposal.
+  const { give } = seat.getProposal();
+  const [[_, amount]] = Object.entries(give);
+
+  // 2. Transfer the tokens into a local account that can initiate the IBC transfer.
+  await zoeTools.localTransfer(seat, sharedLocalAccountP, give);
+
+  // 3. Execute the cross-chain transfer.
+  await sharedLocalAccountP.transfer(
+    { value: destAddr, chainId: (await orch.getChain(chainName)).chainId },
+    { denom, value: amount.value }
+  );
+
+  // 4. Finish. No persistent state is stored for this user.
+  seat.exit();
+};
+```
+
+In this snippet, we perform the following steps:
+
+1. **Extract token Detailss**: The user's seat provides the tokens details (_amount_, _brand_) in a single `give`.
+2. **Send via localAccount**: A local account is used only as a helper to transfer funds from `seat` to an ICA account.
+3. **IBC transfer**: Tokens are transferred cross-chain in a single step via [`transfer` API](/guides/orchestration/key-concepts#funds-transfer).
+4. **No ongoing portfolio**: Once done, the seat concludes—there is no stored user state.
+
+This is a typical example of a transactional contract. It performs a single
+action (sending tokens) and does not maintain any ongoing user state. The user interacts once, and
+the contract concludes the operation. This contract is discussed in more detail in the
+[`send-anywhere` contract walkthrough](/guides/orchestration/contract-walkthroughs/send-anywhere).
+
+## Portfolio Contracts
+
+**Portfolio contracts** manage long-lived user state and support multiple interactions over time.
+They resemble "vaults": you open a portfolio (or vault) position, then manipulate or monitor it as
+desired.
+
+**Characteristics**:
+
+- **Persistent per-user state**: Each user typically has their own account.
+- **Multiple user actions**: The state can be updated with each new user action.
+- **Orchestration Flows**: Users often interact through orchestration flows.
+- **Onboarding**: Users first deposit tokens into a personal portfolio for management.
+
+### Example: "Auto Stake It" Contract
+
+The `auto-stake-it` contract demonstrates how to continuously manage a user's positions. A user
+"onboards" into the contract by creating accounts on multiple chains (via orchestration) and then
+depositing tokens to be staked automatically.
+
+Key points in code:
+
+1. **Prepare a PortfolioHolder** that stores each user's accounts in a single structure.
```

Comment:
The code here doesn't include the prepare step.

```suggestion
1. **Make a PortfolioHolder** that stores each user's accounts in a single structure.
```

To "prepare" is to set up the PortfolioHolder `exoClass` (_I think._). When making an instance, just say "make".